template/.eslintrc.js

```diff
@@ -7,6 +7,14 @@ module.exports = {
   },
   extends: ['@react-native-community'],
```

Comment:
I'm having some weird issues with this line here. Even when I try to install react-native-typescript-template I have this issue: 
![image](https://user-images.githubusercontent.com/55203625/143832875-09e48212-418f-4d7c-8741-d3f5e94a2198.png)

Reply:
Using `yarn install` instead of `npm install` fixes the problem. 🤔

Reply:
Huh...I'm definitely all yarn (it is so. much. faster.) but I do support npm, I had no intention of breaking that, just that I don't test it. That is potentially troublesome. I suppose it's a check to add in CI :laughing:

Reply:
According to https://github.com/facebook/react-native/issues/32181 using `yarn` instead of `npm` does not fix the problem. It just fixed it for me.